esss_fix_format/cli.py

```diff
@@ -6,9 +6,27 @@
 import click
 import subprocess
 
+
+# This is a complete list of all modules in our stdlib which are not already known to isort
+# This is a workaround for https://github.com/timothycrosley/isort/issues/464
+all_stdlib_modules = ["Bastion", "CGIHTTPServer", "DocXMLRPCServer", "HTMLParser", "MimeWriter", "SimpleHTTPServer",
```

Comment:
No, but I don't think it's necessary because the problem only manifests itself on Windows; on Linux the path used matches the logic isort uses.